### ISSUE TITLE
Disable create queue if not exists by configuration

### ISSUE THREAD

**Issue by miranovy** [corvus-ch/rabbitmq-cli-consumer#36]
It is possible to add configuration parameter for disable automatically create queue if not exists? Thanks a lot.

**Comment by corvus-ch** [comment]
No, there is no such parameter or configuration available. @miranovy can you please describe your use case as I have the feeling that ther is a feature request hidden in your question.

**Comment by miranovy** [comment]
We have two separate systems. One produce messages to rabbitmq, second consume them. For consumers we used rabbitmq-cli-consumer. We want to create a queue just from froducees (from one place). Therefore, we would need to disable creating queues from rabbit-cli-consumer by configuration.

Define two status by configuration:
1. auto setup was enabled (default option, current state)
  - queue not exists, rubbitmq-cli-cosumer create it
2. auto setup was disabled (by configuration value)
  - queue not exists, rubbitmq-cli-cosumer write error and exit

Simply put, suppress calling the [setup](https://github.com/corvus-ch/rabbitmq-cli-consumer/blob/master/consumer/consumer.go#L52) method in consumer.go by configuration parameter it is possible.

Thank you very much

**Comment by corvus-ch** [comment]
@miranovy A new release is available containing this change (https://github.com/corvus-ch/rabbitmq-cli-consumer/releases/tag/2.3.0).

**Comment by miranovy** [comment]
Thank you very much, it works great!
